### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

File: .github/workflows/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[*.yml]
+indent_size = 2
```
Comment (peterbourgon) [inline .github/workflows/.editorconfig]: rm please :)

**Review by peterbourgon — APPROVED** [review]
Yeah cool! What's next?

**Comment by sagikazarmark** [comment]
@peterbourgon I initiated a repository transfer to you since I don't have permissions to transfer a repository to the go-kit organization. You should then be able to transfer it to the go-kit organization.

**Comment by peterbourgon** [comment]
Strange! Do you know how I might fix that?

**Comment by sagikazarmark** [comment]
I'm not a part of the go-kit org, so I can't move a repo there. But I was able to initiate a transfer to your personal account requiring your approval (I think you should have received an email).

From there, you should be able to move it to the go-kit org.

**Comment by peterbourgon** [comment]
You are now! Can you try again?

**Comment by sagikazarmark** [comment]
Er... something is not right. GitHub says:

>  You don’t have the permission to create public repositories on go-kit 

🤷

**Pull request merged**